source/common/sequencer_impl.cc

```diff
@@ -83,48 +82,43 @@ void SequencerImpl::updateStartBlockingTimeIfNeeded() {
 void SequencerImpl::run(bool from_periodic_timer) {
   ASSERT(running_);
   const auto now = last_event_time_ = time_source_.monotonicTime();
-  const auto running_duration = now - start_time_;
-
-  // The running_duration we compute will be negative until it is time to start.
-  if (running_duration >= 0ns) {
```

Comment:
Similarly here, can you help me understand the history behind this evaluation and why it is no longer necessary?